packages/react-component-library/src/components/Popover/Popover.tsx

```diff
@@ -41,6 +41,10 @@ export interface PopoverProps
    * Type of component to display (style varies accordingly).
    */
   scheme?: FloatingBoxSchemeType
+  /**
+   * Brief description of the Popover content to aid supportive technologies.
+   */
+  ariaLabel?: string
```

Comment:
Hmm `PopoverProps` is extending `FloatingBoxProps` which has an explicit `aria-label` and `aria-labelledby`. Should we avoid the prop duplication?

Reply:
Nice spot!